src/System.Management.Automation/engine/hostifaces/ChoiceDescription.cs

```diff
@@ -19,6 +19,10 @@ namespace System.Management.Automation.Host
     public sealed
     class ChoiceDescription
     {
+        // DO NOT REMOVE OR RENAME THESE FIELDS - it will break remoting compatibility with Windows PowerShell compatibility with Windows PowerShell
```

Comment:
What about putting these fields within a region to make the grouping more explicit?

Reply:
Good idea.  Will make an update.